src/locale/languages/en.js

```diff
@@ -135,6 +135,7 @@ export const en = {
       allButton: 'All',
       retrievingMessage: 'Retrieving notifications',
       noneMessage: "You don't have any notifications of this type",
+      markAllAsRead: 'Mark all notifications as read',
```

Comment:
This sentence is too long for a button, + translations may be bigger.
Can we use "Mark all read" instead?

Reply:
The size of the button allows, so I made a detailed description, well I'll remove the word "notifications".